generators/README.md

```diff
@@ -4,9 +4,22 @@ This project contains experimental code to create a tool for generating Tekton s
 
 See [tektoncd/pipeline/#2590](https://github.com/tektoncd/pipeline/issues/2590) information and background.
 
+## Cluster Tasks
```

Comment:
We probably want to rephrase this, since there is actually something called a ClusterTask, and that's not a requirement for what we're doing here - https://pkg.go.dev/github.com/tektoncd/pipeline@v0.14.0/pkg/apis/pipeline/v1beta1?tab=doc#ClusterTask